### PR TITLE
Render markdown preview

### PR DESCRIPTION
**Pull request by dmrib**
Depends on #5 

First commit is: `Add marked.js as a dependency`

### CODE REVIEW

File: linguicator-editor/src/components/Content/Content.js [hunk 1]
```diff
@@ -1,15 +1,59 @@
...
 {
+    // component state hooks
+    const [text, setText] = useState('');
+    const [preview, setPreview] = useState('');
+
+
+    /**
+     * I am called whenever text input is changed and set sanitized markdown preview.
+     *
+     * Returns:
```
Comment (HeitorBoschirolli) [inline linguicator-editor/src/components/Content/Content.js]: what about using jsdoc to document the functions? Seems to be the most popular choice.

**Review by HeitorBoschirolli — CHANGES_REQUESTED** [review]
Also missing the version update.

**Comment by HeitorBoschirolli** [comment]
Didn't quite got the difference between this PR and [this one](https://github.com/machinegrooving/linguicator-editor/pull/8/)

**Comment by HeitorBoschirolli** [comment]
+1

File: linguicator-editor/src/components/Content/Content.js (hunk 1 above)
Comment (dmrib) [reply linguicator-editor/src/components/Content/Content.js]: Done.

**Review by HeitorBoschirolli — APPROVED** [review]
+1

**Pull request merged**